ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewApplicationTest.java

```diff
@@ -0,0 +1,74 @@
+        System.out.println("Check that correct Home page (after login) is opened");
+        Assert.assertEquals(driver.getTitle(), "Recently Viewed | Applications | Salesforce", "Incorrect Home page is not opened");
+        WebElement alicationDropdown = driver.findElementByXPath("//span[text()='Applications']/../../one-app-nav-bar-item-dropdown//*[@role='button']");
+        alicationDropdown.click();
+        System.out.println("Find and click Applications dropdown");
+        WebElement createNewAppButton = driver.findElementByXPath("//span[text()='New Application']");
+        createNewAppButton.click();
+        System.out.println("Find and click New Application option");
+        WebElement formTradeNameInput = driver.findElementById("input-3");
+        formTradeNameInput.sendKeys("yauhen0707" + x);
+        System.out.println("Find and populate Trade Name field");
+        WebElement formEmailInput = driver.findElementByName("input-12");
+        formEmailInput.sendKeys("yauhen" + x + "@example.com");
+        System.out.println("Find and populate Merchant Email field");
+        WebElement formNumberOfAdditionalOutletsInput = driver.findElementById("input-01");
+        formNumberOfAdditionalOutletsInput.sendKeys("1");
+        System.out.println("Find and populate Number of additional outlets field");
+        WebElement formBusinessNatureDropdown = driver.findElementByXPath("//div/select/option[@value='Computer Services']");
+        formBusinessNatureDropdown.click();
+        System.out.println("Find and select value from Business Nature dropdown");
+        WebElement formNextButton = driver.findElementByXPath("//button[text()='Next']");
+        formNextButton.click();
+        System.out.println("Find and click Next button");
+        WebElement secontPartFormTitle = driver.findElementByXPath("//h2[@class=\"title slds-text-heading--medium\"]");
+        System.out.println("Check that second part of form Create new Application is opened");
+        String secontPartFormTitleValue = secontPartFormTitle.getText();
+        Assert.assertEquals(secontPartFormTitleValue, "New Application: New", "Second part of Create new Application form is not opened");
+        WebElement formLastNameInput = driver.findElementByXPath("//input[@maxlength='80' and @data-interactive-lib-uid='24']");
+        formLastNameInput.sendKeys("test");
+        System.out.println("Find and populate Last Name field");
+        WebElement formBusinessLineDropdown = driver.findElementByXPath("//span[text()='Business Line']/../..//a[@class='select']");
+        formBusinessLineDropdown.click();
+        System.out.println("Find and open Business Line drop down");
+        WebElement formBusinessLineValue = driver.findElementByXPath("//a[text()='Computer Network/Information Services']");
+        formBusinessLineValue.click();
+        System.out.println("Find and select value from Business drop down");
```

Comment:
add assertion that Computer Network/Information Services text is displayed

Reply:
Done